### ISSUE TITLE
Prepare Content Method

### ISSUE THREAD

**Issue by jfrank-nih** [NCIOCPL/akumina-mynci#23]
## Summary

Create a `prepareContent` method in the `DrupalContent` class that will convert an arbitrary JavaScript object into a new JavaScript object able to be loaded in to SharePoint (matching the PNPJS expectations for content). It should be able to apply specified transformations (to be separately developed) to the data.

## Inputs

- An array that contains objects that we want to load into SharePoint from the `loadContent` method #12. Can be arbitrary:

```node
[
{
  path: {
    to: {
      key: {
        in: {
          object: "Something here!"
        }
      },
      otherkey: {
        in: {
          object: "Something here too!"
        }
      },
      yetanotherkey: {
        in: {
          object: "And more!"
        }
      }
    }
  }
},
...
]
```

- A translation map between the object and the SharePoint column name you want to map to. Exact format can be determined, but something like this:

```node
[
{
   SharePointColumn: "Column1",
   ContentPaths: [
        "path.to.key.in.object"
    ],
   Transformation: "convertText"
},
{
   SharePointColumn: "Column2",
   ContentPaths: [
        "path.to.otherkey.in.object",
        "path.to.yetanotherkey.in.object"
    ],
    Separator: "<br />"
},
...
]
```

## Outputs

- A new array of transformed objects that can be passed to `importList` #16

Basically we want to take an arbitrary array of objects from `loadContent` and transform it into a new array:

```
[
{
   "Column1": "Something here!"
   "Column2": "Something here too!<br /><br />And more!"
},
...
]
```

_Tasks:_

- Create a new, empty result array
- Loop through all of the objects in the content array
- For each object in the array, transform it into a new object with keys that match the translation map and values that derive from the input object
  - If there is more than one value specified from the input object then concatenate all the different values together in order using the specified `Separator`
- We will return eventually to add transformations on, but don't need to in this ticket
